### ISSUE TITLE
win32process.CreateRemoteThread Does not allow python strings

### ISSUE THREAD

**Issue by ughstudios** [mhammond/pywin32#1890]
win32process.CreateRemoteThread seems to not take python strings. 

`thread_handle, thread_id = win32process.CreateRemoteThread(handle, None, 0, remote_load_library, library_name, 0)`

Running the following line of  code, results in a type error of: `Unable to convert str to pointer-sized value` 

library_name is the string in question, its just `library_name = 'mydll.dll'`

**Comment by ughstudios** [comment]
Here's where the code is for this: https://github.com/mhammond/pywin32/blob/f7f12ec98de599f5afb62a5ba467b17a63cd660a/win32/src/PyWinTypesmodule.cpp#L534

**Comment by ughstudios** [comment]
Nevermind, after reading the code more thoroughly I realized that the args cannot be a string, they must be a void pointer.

**Comment by mhammond** [comment]
I'd be happy to take a PR which improves the docs here, or even better, has a very quick example - but for now I'm going to close this.

**Comment by ughstudios** [comment]
Hi @mhammond I figured out what was wrong. CreateRemoteProcess must take a block of memory. 

```
thread_handle, thread_id = win32process.CreateRemoteThread(handle, None, 0, remote_load_library, memory_address, 0)
```

For example, "memory_address" was created by allocating virtual memory in another process. 
```
memory_address = win32process.VirtualAllocEx(handle, 0, len(library_name), win32con.MEM_COMMIT | win32con.MEM_RESERVE, win32con.PAGE_READWRITE)
```

```
win32process.WriteProcessMemory(handle, memory_address, library_name)
```

I'm not sure if this is necessary for the memory to exist on the "remote process" or if it just needs to be a memory block. But this works well for me. 

I allocated virtual memory on the other process using VirtualAllocEx() and then wrote to it using WriteProcesssMemory() which took my python `library_name` string. Then I passed that memory_address that was allocated into CreateRemoteThread
